lighthouse-core/test/fraggle-rock/gather/navigation-runner-test.js

```diff
@@ -97,22 +104,39 @@ describe('NavigationRunner', () => {
 
     it('should navigate to the blank page', async () => {
       await runner._setup({driver, config, requestedUrl});
-      expect(mockDriver._page.goto).toHaveBeenCalledWith('about:blank');
+      expect(mocks.navigationMock.gotoURL).toHaveBeenCalledTimes(1);
+      expect(mocks.navigationMock.gotoURL).toHaveBeenCalledWith(
+        expect.anything(),
+        'about:blank',
+        expect.anything()
+      );
     });
 
-    it.todo('should throw if service worker is currently controlling the page');
-    it.todo('should enable emulation');
-    it.todo('should enable important CDP domains');
-    it.todo('should register the performance observer for navigation conditions');
-    it.todo('should shim requestIdleCallback');
-    it.todo('should reset storage');
-    it.todo('should not reset storage when skipped');
```

Comment:
The reset storage tests seem useful at least.

Reply:
agreed, but these are all tested in `prepare-test.js` now, I put these placeholders here when the runner was first added to capture what gather-runner did at the time, and didn't update as those pieces were stripped out of gather-runner into submodules.

Or were you commenting that an integration test where the preparation module isn't mocked would be preferred?

Reply:
I was thinking these applied to the code in `_cleanup` as well.

Reply:
Ah gotcha, for `_cleanup`! Yes good catch 👍